app/models/npq_application.rb

```diff
@@ -94,6 +110,10 @@ def self.participant_declaration_finder(participant_identity_id)
     ParticipantDeclaration::NPQ&.find_by_participant_profile_id(ParticipantProfile&.find_by_participant_identity_id(participant_identity_id)&.id)
   end
 
+  def declared_as_payable?
+    profile.participant_declarations.paid_payable_or_eligible.count.positive?
```

Comment:
could we use the `billable` scope here? also maybe we can rename the method to `with_billable_declarations?` as it might be more descriptive?

Reply:
Sure, Just out of curiosity, why we have `billable` and `paid_payable_or_eligible` scopes?